docs/README.md

```diff
@@ -1 +1,13 @@
+# Orchestration documentation
+
 This catalog describes how the orchestration works in the current state
+
+### How to generate dag svg files
+
+1. Locate your global `airflow.cfg` file and update the [core] dag_folder in `airflow.cfg` to point to the `src` directory of the orchestration repository or set the `AIRFLOW__CORE__DAGS_FOLDER` environment variable.
+
+2. Run
+
+   ```bash
```

Comment:
It would be cool to do this either in CI/CD or when building documentation (if we go that way)

Reply:
It's definately something we should do, but I do not know if building it automatically will not cause more trouble if we keep the reference to the files in the individual datasource docs. We would have to map them somehow? Lets keep it for the future.